src/components/TransformWorkflow/FlowEditor/Components/OperationPanel/Forms/CastColumnOpForm.tsx

```diff
@@ -18,6 +11,7 @@ import Input from '@/components/UI/Input/Input';
 import { GlobalContext } from '@/contexts/ContextProvider';
 import { errorToast } from '@/components/ToastMessage/ToastHelper';
 import { OperationFormProps } from '../../OperationConfigLayout';
+import { Autocomplete } from '@/components/UI/Autocomplete/Autocomplete';
 
 const castGridStyles: {
   container: SxProps;
```

Comment:
@mdshamoon the grid in the cast operation is a bit off (cells in both columns are not in the same row). Also the left side column input is not using the new `Input` component you have designed. Is that on purpose ?

Reply:
@Ishankoradia was planning to take the whole grid separately as it has somewhat different design elements than normal autocomplete and input.